README.md

```diff
@@ -52,7 +52,7 @@ Features:
 
 ## Why?
 
-When I need to call an HTTP API, I usually need to add dynamic parameters to the URL:
+When we call an HTTP API, we usually need to add dynamic parameters to the URL:
```

Comment:
I wrote this section in the first person because I want the README to be more personal. To be honest I would want to keep it that way; still, thanks for taking the time to read through it.